docs/SPEC.md

```diff
@@ -1333,35 +1317,12 @@ Examples using known start and stop dates:
     // [2018-03-15, 2018-04-15)
     // [2018-04-15, 2018-05-15)
 
-
-[IMPL#659](https://github.com/influxdata/platform/issues/659) Implement intervals function
-
-
-### Builtin Intervals
-
-The following builtin intervals exist:
-
-    // 1 second intervals
-    seconds = intervals(every:1s)
-    // 1 minute intervals
-    minutes = intervals(every:1m)
-    // 1 hour intervals
-    hours = intervals(every:1h)
-    // 1 day intervals
-    days = intervals(every:1d)
-    // 1 day intervals excluding Sundays and Saturdays
-    weekdays = intervals(every:1d, filter: (interval) => weekday(time:interval.start) not in [Sunday, Saturday])
-    // 1 day intervals including only Sundays and Saturdays
-    weekdends = intervals(every:1d, filter: (interval) => weekday(time:interval.start) in [Sunday, Saturday])
-    // 1 week intervals
-    weeks = intervals(every:1w)
-    // 1 month interval
-    months = intervals(every:1mo)
-    // 3 month intervals
-    quarters = intervals(every:3mo)
-    // 1 year intervals
-    years = intervals(every:1y)
```

Comment:
Hmm should we add these to an `intervals` package?

Reply:
If we decide to keep the intervals function, then yeah I think that would be convenient. I can create an issue to add these functions in and document them.